app/javascript/orangelight/availability.es6

```diff
@@ -25,10 +25,11 @@ export default class AvailabilityUpdater {
     this.process_barcodes = this.process_barcodes.bind(this);
     this.process_single = this.process_single.bind(this);
     this.update_single = this.update_single.bind(this);
+    this.update_single_undetermined = this.update_single_undetermined.bind(this);
     this.process_scsb_single = this.process_scsb_single.bind(this);
   }
 
-  request_availability() {
+  request_availability(allowRetry) {
```

Comment:
In the future, we could make this a number of retries (rather than a boolean) but for now a single retry should be OK.